docs/USER_GUIDE.md

```diff
@@ -52,10 +52,17 @@ Interfacing linear code with regular Haskell is done, for instance, through the
 The data type `Ur`, short for _unrestricted_ lets you store an
 unrestricted value inside a linear value.
 
+### Import Conventions
+
+We've designed `linear-base` to work nicely with the following import conventions:
+
+1. `import qualified Data.Functor.Linear as Data`
+2. `import qualified Control.Functor.Linear as Control`
```

Comment:
Since it's not actually an ordered list, it may be preferable to use bullets instead

```suggestion
- `import qualified Data.Functor.Linear as Data`
- `import qualified Control.Functor.Linear as Control`
```